homeassistant/components/camera/__init__.py

```diff
@@ -98,6 +99,18 @@ class Image:
     content = attr.ib(type=bytes)
 
 
+@bind_hass
+async def async_request_stream(hass, entity_id, format):
```

Comment:
Should we pass through the optional kwargs here as well and keep this signature similar to `request_stream` in the stream component?

Reply:
Let's add it when we have a use case.